test/sampler_test.rb

```diff
@@ -79,15 +79,15 @@ def test_tracer_with_rate_sampler
     prng = Random.new(123)
 
     tracer = get_test_tracer()
-    tracer.configure(sampler: Datadog::RateSampler.new(0.5))
+    tracer.configure(sampler: Datadog::RateSampler.new(0.5), priority_sampling: false)
```

Comment:
Not sure if adding this is right: problem is if you leave it to the default, the `RateSampler` ends up wrapped with a `PrioritySampler` and that causes it to break the `0.5` rate expectation below. Only by disabling priority sampling do we get this to pass.

Reply:
yeah, I mean it is how we would get a `RateSampler` since by default now we are using `PrioritySampler`